src/pages/patterns/dialog-pattern/index.mdx

```diff
@@ -84,11 +84,11 @@ When a _modeless_ dialog is active the user can continue viewing and interacting
 
 #### Use dialogs sparingly
 
-Don't overuse dialogs. They are disruptive and can easily annoy the user if used incorrectly or too frequently. When dialogs are used for non-workflow related tasks, it is likely a user will start ignoring or dismissing the dialogs without fully understanding the content. This can cause users to make hurried or impulsive choices when dealing with more critical dialogs.
+Don’t overuse dialogs. They are disruptive and can easily annoy the user if used incorrectly or too frequently. When dialogs are used for non-workflow related tasks, it is likely a user will start ignoring or dismissing the dialogs without fully understanding the content. This can cause users to make hurried or impulsive choices when dealing with more critical dialogs.
```

Comment:
Just out of curiosity, aren't these converted into smart quotes in body copy when the site builds? @vpicone @alisonjoseph

Reply:
Anything in markdown should be, yes. Quotes inside custom components have to be typed manually.

Reply:
Yeah the ones in the captions weren't converting and I just replaced the other ones I saw as well for good measure.

Reply:
Thanks @alisonjoseph! Just wanted to be sure I wasn't dropping the ball over here.